package.json

```diff
@@ -128,5 +128,9 @@
     "hooks": {
       "pre-push": "node tools/gitHooks/prepush.js"
     }
+  },
+  "dependencies": {
```

Comment:
This should be a dev-dependency in packages/firestore.

Reply:
Done.